docs/examples/ethane_box_oplsaa.ipynb

```diff
@@ -0,0 +1,69 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mbuild as mb\n",
+    "from mbuild.examples import Ethane\n",
+    "from foyer.tests.utils import get_fn\n",
+    "from foyer import Forcefield"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Approach 1 ###\n",
```

Comment:
Could we have a more descriptive name? 
```suggestion
### Applying a force field while saving from mBuild
```

Reply:
Just leaving a note, we should update this in the paper as well then.  The point here was to address the reviewer comment that these examples from the paper weren't available on github.